src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -450,6 +450,70 @@ def test_backup_protection(self, resource_group, vault_name, vm_name):
         protection_check = self.cmd('backup protection check-vm --vm-id {vm_id}').output
         self.assertTrue(protection_check == '')
 
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location="southeastasia")
+    @VaultPreparer(soft_delete=False)
+    @VMPreparer()
+    @ItemPreparer()
+    @RPPreparer()
+    @live_only()
```

Comment:
Actually, this test is getting used for testing cross-subscription scenario. Running the recording fails to find the subscription probably because the subscription gets recorded as zeros. Therefore, I had to add the live_only() tag.

![MicrosoftTeams-image](https://user-images.githubusercontent.com/26112239/171369805-7f3f87c2-993d-487e-9b2e-fb693c140085.png)